lib/ClimaLandSimulations/src/utilities/met_drivers_FLUXNET.jl

```diff
@@ -198,9 +198,8 @@ function setup_drivers(site_ID)
     LAI_dt = Second(MODIS_LAI[2, 1] - MODIS_LAI[1, 1]).value
     LAI_seconds = FT.(0:LAI_dt:((length(MODIS_LAI[:, 1]) - 1) * LAI_dt))
 
-    # LAI spline for radiative transfer
-    LAIspline = Spline1D(LAI_seconds, MODIS_LAI[:, 2])
-    LAIfunction = (t) -> eltype(t)(LAIspline(t))
+    # LAI function for radiative transfer
+    LAIfunction = TimeVaryingInput(LAI_seconds, FT.(MODIS_LAI[:, 2]); context)
```

Comment:
sorry that you had to do this twice! we are keeping the experiments folder around/as is while this is being built up...